app.js

```diff
@@ -3,14 +3,15 @@ const app=express();
 
 const {getTopics, 
     getArticles,
-    }
+    getArticleId}
```

Comment:
Realy like that you're following a consistent naming pattern for your models and controllers